server/routes/search/index.ts

```diff
@@ -1,13 +1,53 @@
-import { Router } from 'express'
+import { NextFunction, Request, Response, Router } from 'express'
 import { Services } from '../../services'
 import asyncMiddleware from '../../middleware/asyncMiddleware'
 import SearchController from './searchController'
+import SearchSortField from '../../enums/searchSortField'
+import SearchSortDirection from '../../enums/searchSortDirection'
+import config from '../../config'
+import { Result } from '../../utils/result/result'
 
-const searchRoutes = (_services: Services): Router => {
+const DEFAULT_SORT_FIELD = SearchSortField.PRISONER_NAME
+const DEFAULT_SORT_DIRECTION = SearchSortDirection.ASC
+
+const searchRoutes = (services: Services): Router => {
   const router = Router({ mergeParams: true })
+  const { searchService } = services
   const searchController = new SearchController()
 
-  router.get('/', [asyncMiddleware(searchController.getSearchView)])
+  const performSearch = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I've opted to perform the actual search here in the route, as a middleware; rather than in the controller method.
This approach keeps the controller really slim and lightweight, and (if I refactored this method out to a standalone middleware function) it becomes testable and re-useable (not sure if there is a use case to warrant that yet 🤔 )